HW4/Task3.py

```diff
@@ -0,0 +1,14 @@
+print("Homework 4. Task 3. Make list of unique numbers from given list")
+print()
+
+lst = [5, 5 , 12, 0, 1, 12, 6, 5, 3]
+result_lst = []
+
+for i in lst:
```

Comment:
Сложность операций с циклами – линейная, но у вас внутри цикла используется метод `count`, сложность которого также линейная. Сами понимаете, как общая сложность увеличивается. Однако можно немножко спасти ситуацию таким образом:
```suggestion
for i in set(lst):
```
Однако, это не сильно поможет, если в исходном списке нет повторяющихся элементов.

Reply:
Почему же не поможет? В результат запишутся все элементы, которые по сути уникальны до единого.

Reply:
Верно, но я говорил о сложности. Сложность при этом будет максимальная, не важно, используем ли мы множество в объявлении цикла или нет.